packages/valory/skills/proposal_collector_abci/rounds.py

```diff
@@ -72,9 +72,9 @@ def proposals(self) -> dict:
         return cast(dict, self.db.get("proposals", {}))
 
     @property
-    def selected_proposal_id(self) -> str:
-        """Get the selected_proposal_id."""
-        return cast(str, self.db.get_strict("selected_proposal_id"))
+    def votable_proposal_ids(self) -> list:
+        """Get the proposals."""
+        return cast(list, self.db.get("votable_proposal_ids", {}))
```

Comment:
```suggestion
    def votable_proposal_ids(self) -> set:
        """Get the proposals."""
        return set(self.db.get("votable_proposal_ids", {}))
```

Reply:
Agree, feel free to commit.

Reply:
Done in e7c127968905b1febeede71ad01f894b8c324cc7.